lib/session.d.ts

```diff
@@ -0,0 +1,34 @@
+declare module 'telegraf-session-local' {
+  import { AdapterSync, AdapterAsync, BaseAdapter } from 'lowdb'
+  import { ContextMessageUpdate, Middleware } from 'telegraf'
+
+  interface LocalSessionOptions {
+    storage?: AdapterSync | AdapterAsync
+    database?: string
+    property?: string
+    state?: object
+    format?: object
+    getSessionKey?: (ctx: ContextMessageUpdate) => string
+  }
+
+  class LocalSession {
+    public DB: AdapterSync | AdapterAsync
+
+    constructor(options?: LocalSessionOptions)
+
+    getSessionKey(ctx: ContextMessageUpdate): string
+    getSession(key: string): object
```

Comment:
Yeah, you absolutely right, I just needed a fast solution for myself and since this repo has no typings I posted mine to have at least something to start from.

Reply:
I suggest having `LocalSessionState` as an additional generic interface, so developers can Have their `session` of any type, but at the same type keeping `state` as `object` in `LocalSessionOptions` interface by default.
Checkout updated code.

Reply:
keeping it as `any` can easily done via `LocalSessionState<any>`. I think that is a cleaner way as the user has to actively decide to use less type checking.